app/helpers/hosts_extensions_helper.rb

```diff
@@ -0,0 +1,62 @@
+module HostsExtensionsHelper
+  def hosts_multiple_actions
```

Comment:
I still think we should use `rex_` prefix.
This could in theory clash with methods from other plugins, couldn't it?